<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmSampling</class>
 <widget class="QWidget" name="FrmSampling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbSource">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(220, 220, 220, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
font: 75 10pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>[SOURCE]</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTechnique">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Description</string>
          </property>
          <property name="buddy">
           <cstring>textDescription</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="textDescription"/>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Comments</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="textComments"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Global Strategy</string>
          </property>
          <property name="buddy">
           <cstring>cmbStrategy</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbStrategy"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Sample Size</string>
          </property>
          <property name="buddy">
           <cstring>spinSample</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinSample">
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Population Size</string>
          </property>
          <property name="buddy">
           <cstring>spinPopulation</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinPopulation">
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Levels</string>
            </property>
            <property name="buddy">
             <cstring>tableView</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushInsert">
            <property name="toolTip">
             <string>Insert row</string>
            </property>
            <property name="statusTip">
             <string>Insert row</string>
            </property>
            <property name="whatsThis">
             <string>Inserts row into table</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="app_new.qrc">
              <normaloff>:/app_new/add.png</normaloff>:/app_new/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushRemove">
            <property name="toolTip">
             <string>Remove row</string>
            </property>
            <property name="statusTip">
             <string>Remove row</string>
            </property>
            <property name="whatsThis">
             <string>Removes row into table</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="app_new.qrc">
              <normaloff>:/app_new/cancel.png</normaloff>:/app_new/cancel.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>308</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushBack">
       <property name="toolTip">
        <string>Go back</string>
       </property>
       <property name="statusTip">
        <string>Go back to previous form</string>
       </property>
       <property name="whatsThis">
        <string>Goes back to previous form</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushApply">
       <property name="toolTip">
        <string>Apply changes</string>
       </property>
       <property name="statusTip">
        <string>Apply</string>
       </property>
       <property name="whatsThis">
        <string>Applies changes</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="app_new.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushApply</sender>
   <signal>clicked()</signal>
   <receiver>FrmSampling</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushBack</sender>
   <signal>clicked()</signal>
   <receiver>FrmSampling</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushInsert</sender>
   <signal>clicked()</signal>
   <receiver>FrmSampling</receiver>
   <slot>insertRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushRemove</sender>
   <signal>clicked()</signal>
   <receiver>FrmSampling</receiver>
   <slot>removeRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>entered(QModelIndex)</signal>
   <receiver>tableView</receiver>
   <slot>resizeColumnsToContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>back()</slot>
  <slot>apply()</slot>
  <slot>initRecords(int)</slot>
  <slot>insertRow()</slot>
  <slot>removeRow()</slot>
 </slots>
</ui>
